projects/Mallard/src/AppNavigation.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react'
```

Comment:
This file is currently a catch all for screens. We should consider correctly stacking